src/styles/index.scss

```diff
@@ -1,3 +1,31 @@
+$font-family: Roboto, sans-serif;
+$white_color: #fff;
+$hover_text_color: #34568b;
+$border_color: #f3f3f3;
+$layout-breakpoint-small: 488px;
+$layout-breakpoint-medium: 768px;
+$layout-breakpoint-large: 1024px;
+$card-width: 200px;
+$font-family: Roboto, sans-serif;
+$color-title: #060b35;
+$color-code: #616070;
+$color-buy: #00acdc;
+$color-price: #616070;
```

Comment:
- Let's create a separate file with variables.
- There's no need for separate variables if the same value is used multiple times. Variables are used to store values that might change, so if the same value is used in multiple places, it's more efficient to use the same variable instead of creating duplicates.  Let's reuse the same variable throughout your files for consistency.
Example: 

```
// Using a variable
$primary-color: #3498db;

.button {
  background-color: $primary-color;     // we use variable here
  color: darken($primary-color, 10%);  // and same variable here
}
```